src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -36,10 +39,32 @@ public ComputedAttributesHandler() {
         engine.setStrict(true);
     }
 
+    private MapContext prepareContext(Position position) {
+        MapContext result = new MapContext();
+        Method[] methods = position.getClass().getMethods();
+        for (Method method : methods) {
+            if (method.getName().startsWith("get") && method.getParameterTypes().length == 0
+                    && !method.getName().equals("getClass")) {
+                String name = method.getName().substring(3, 4).toLowerCase() + method.getName().substring(4);
```

Comment:
You can do a small optimization here and use `Character.toLowerCase(...)` method on a single `char`.